README.md

```diff
@@ -1,2 +1,126 @@
-# sample_alert_info
-Release notes for sample Rubin Observatory alert packets
+# Rubin Observatory Sample Alerts
+
+The Vera C. Rubin Observatory is releasing sample alert packets produced by its pipelines run on precursor survey data.
+These alerts are intended to provide candidate community alert brokers as well as science users the chance to preview the alert format and contents and to provide feedback to the project.
+
+## Obtaining the Data
+
+Sample alert data may be obtained at [this link](https://lsst.ncsa.illinois.edu/~ebellm/sample_precursor_alerts/).
+We anticipate providing reprocessed and updated alert samples as pipeline development progresses, so the data are arranged in a directory structure:
+
+```
+    sample_precursor_alerts/
+	YYYY-MM-DD/
+	    dataset/
+		all_visits_{dataset}_{date}.avro
+		association_{dataset}_{date}.db
+		single_ccd_sample_{dataset}_{date}.avro
+		single_visit_sample_{dataset}_{date}.avro
+```
+
+### Alert packets
+
+Alerts are provided as Snappy-compressed [Avro](https://avro.apache.org/) files.
+The schema used to store the alerts is included in the file.
+
+As the complete sample is about 1 TB, We include two smaller subsets--alerts from a single CCD-visit, and all of the alerts from all CCDs in a single visit.
+
+### Database
+In addition to accessing alert packets via community brokers, during the LSST survey science users with data rights will be able to query the relational Prompt Products Database (PPDB).  
```

Comment:
It's still Apdb correct? If so you've got more than a few of these to change name wise.

Reply:
End users will access the PPDB rather than the APDB, so I used it deliberately throughout.